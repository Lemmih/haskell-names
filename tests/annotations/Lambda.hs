module Lambda where

f = \x y -> x y
