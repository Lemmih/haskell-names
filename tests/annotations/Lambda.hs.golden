x @ tests/annotations/Lambda.hs: (3:13) - (3:14)
LocalValue {sLoc = (), sDefLoc = SrcLoc {srcFilename = "tests/annotations/Lambda.hs", srcLine = 3, srcColumn = 6}}
y @ tests/annotations/Lambda.hs: (3:15) - (3:16)
LocalValue {sLoc = (), sDefLoc = SrcLoc {srcFilename = "tests/annotations/Lambda.hs", srcLine = 3, srcColumn = 8}}
